Source/WebKit/Configurations/BaseExtension.xcconfig

```diff
@@ -27,33 +27,29 @@ CLANG_ENABLE_OBJC_ARC = YES;
 CLANG_ENABLE_MODULES = YES;
 CLANG_INSTRUMENT_FOR_OPTIMIZATION_PROFILING = NO;
 INSTALL_PATH = $(WK_INSTALL_PATH_PREFIX)/System/Library/ExtensionKit/Extensions;
-SWIFT_OBJC_BRIDGING_HEADER = $(SWIFT_OBJC_BRIDGING_HEADER_$(USE_INTERNAL_SDK));
-SWIFT_OBJC_BRIDGING_HEADER_ = ;
-SWIFT_OBJC_BRIDGING_HEADER_YES = ;
-SWIFT_OBJC_BRIDGING_HEADER_YES[sdk=iphone*] = Shared/AuxiliaryProcessExtensions/AuxiliaryProcessExtensionBridge.h;
-SWIFT_OBJC_BRIDGING_HEADER_YES[sdk=iphone*17.0*] = ;
-SWIFT_OBJC_BRIDGING_HEADER_YES[sdk=iphone*17.1*] = ;
-SWIFT_OBJC_BRIDGING_HEADER_YES[sdk=iphone*17.2*] = ;
-SWIFT_OBJC_BRIDGING_HEADER_YES[sdk=iphone*17.3*] = ;
-SWIFT_OBJC_BRIDGING_HEADER_YES[sdk=iphone*18.0*] = ;
-SWIFT_OBJC_BRIDGING_HEADER_YES[sdk=xr*] = ;
+SWIFT_OBJC_BRIDGING_HEADER = ;
+SWIFT_OBJC_BRIDGING_HEADER[sdk=iphone*] = Shared/AuxiliaryProcessExtensions/AuxiliaryProcessExtensionBridge.h;
+SWIFT_OBJC_BRIDGING_HEADER[sdk=iphone*17.0*] = ;
+SWIFT_OBJC_BRIDGING_HEADER[sdk=iphone*17.1*] = ;
+SWIFT_OBJC_BRIDGING_HEADER[sdk=iphone*17.2*] = ;
+SWIFT_OBJC_BRIDGING_HEADER[sdk=iphone*17.3*] = ;
```

Comment:
Possibly a dumb question, but does this need this line somewhere?
`SWIFT_OBJC_BRIDGING_HEADER[sdk=iphone*17.4*] = ;`